README.md

```diff
@@ -21,6 +21,20 @@ It uses:
 ### Plugins
 * [lombok plugin](https://plugins.jetbrains.com/idea/plugin/6317-lombok-plugin) - Lombok IDEA plugin
 
+#### Environment variables
+The following environment variables are required:
+
+| Name | Default | Description |
+|------|---------|-------------|
+| IDAM_USER_BASE_URI | - | Base URL for IdAM's User API service (idam-app). `http://localhost:4501` for the dockerised local instance or tunneled `dev` instance. |
+| IDAM_S2S_BASE_URI | - | Base URL for IdAM's S2S API service (service-auth-provider). `http://localhost:4502` for the dockerised local instance or tunneled `dev` instance. |
+| MAX_FILE_SIZE | 500MB | Max file size |
```

Comment:
What was the reason for setting `MAX_FILE_SIZE` to 500MB while the `maxAllowedContentLength ` is 100MB?  Is it to leave sleeping dogs lie in other parts of the code maybe?

Reply:
Well I wanted the number of file changes to be minimum and someone to spot this variation 😁 (easy one).  More files for you to review now, changed default 500MB to 100MB